eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelData.scala

```diff
@@ -466,7 +466,7 @@ case class LocalParams(nodeId: PublicKey,
                        fundingKeyPath: DeterministicWallet.KeyPath,
                        dustLimit: Satoshi,
                        maxHtlcValueInFlightMsat: UInt64, // this is not MilliSatoshi because it can exceed the total amount of MilliSatoshi
-                       channelReserve: Satoshi,
+                       requestedChannelReserve: Satoshi,
```

Comment:
You can probably make that an `Option` and update the codec to have it take a `channelFeatures` in parameter.

Reply:
Good idea! Done in https://github.com/ACINQ/eclair/pull/2237/commits/0bcf7a089b35765d8425f0ad7987b6a237e241fc